src/main/java/com/marginallyclever/makelangelo/select/Select.java

```diff
@@ -25,6 +30,13 @@ protected Select(String name) {
 	// OBSERVER PATTERN
 	
 	public void addPropertyChangeListener(PropertyChangeListener p) {
+		if ( propertyChangeListeners == null){ 
+			// PPAC37 to avoid "-nolf" : as some linux Look and Feel (like "com.sun.java.swing.plaf.gtk.GTKLookAndFeel")
+			// can run this overrided methode befor the class is fully initialised ...
+			// (But then tooltips color to change TODO.)
+			propertyChangeListeners = new ArrayList<>();
+			logger.debug("need \"-nolf\" due to l&f \"{}\"",UIManager.getLookAndFeel().getClass().getCanonicalName());
```

Comment:
the log will be full of this trace, maybe you can make only one trace on `Makelangelo.java` at startup

Reply:
you are right, with the boolean, it is not beautiful :-( what do you think of adding the log in makelangelo.java when detecting a linux ?

Reply:
Why not ... but I do not see how to implement this simply (because there must be implementations of LookAndFeel other than the one I quote and on other OS which can have the same result ...) 
 and since there is that at this time 
~~~
public void addPropertyChangeListener(PropertyChangeListener p) {
		if ( propertyChangeListeners == null){ 
~~~
 where I know there is a problem with the current L&F...)

Or maybe I don't understand what you're trying to ask me. (I'm not very fresh today)

Reply:
I see your point, I was thinking of something like this in https://github.com/MarginallyClever/Makelangelo-software/blob/bc9dd294a2d0fed1de1340b3722262f0cbc84be3/src/main/java/com/marginallyclever/makelangelo/Makelangelo.java#L203:
```
if(!CommandLineOptions.hasOption("-nolf")) {
	try {
	    UIManager.setLookAndFeel(UIManager.getSystemLookAndFeelClassName());
	} catch (Exception e) {}
} else if ("com.sun.java.swing.plaf.gtk.GTKLookAndFeel".equals(UIManager.getLookAndFeel().getClass().getCanonicalName())) {
   String msg = "please launch me with the option \"-nolf\" to avoid some graphical bug due to l&f " + UIManager.getLookAndFeel().getClass().getCanonicalName();
   logger.warn(msg);
   System.err.println(msg);
}
```

what do you think ?

Reply:
I'm divided,
your proposal works fine if we assume that "com.sun.java.swing.plaf.gtk.GTKLookAndFeel" will always have a problem and that there are no other L&Fs that have or possibly will have the problem ...

Reply:
For the moment, as there is no user choice to select another L&F, it is the only one that is available for linux, so we can assume this.
A better implementation would be to disable the L&F of gnome if it is detected in order to avoid adding the option `-nolf` to avoid confusion of users:
```
if (!"com.sun.java.swing.plaf.gtk.GTKLookAndFeel".equals(UIManager.getLookAndFeel().getClass().getCanonicalName()) {
	try {
	    UIManager.setLookAndFeel(UIManager.getSystemLookAndFeelClassName());
	} catch (Exception e) {}
} else {
   logger.warn("Look and feel disabled to avoid some graphical bug");
}
```

Better ?